ush/wrappers/job_cards/sbatch/run_post.sbatch

```diff
@@ -0,0 +1,42 @@
+#! /bin/sh
+#SBATCH --account=epic
+#SBATCH -o run_post.log
+#SBATCH --nodes=2-2
+#SBATCH --tasks-per-node=24
+#SBATCH --qos=batch
+#SBATCH -t 00:30:00
+#SBATCH --job-name=run_post
+#SBATCH --export=NONE
+
+# Prepared for Hera mchine
+
+#--- for propper work, make sure of placement of your job card relative to var_defns.sh file ---#
+export GLOBAL_VAR_DEFNS_FP=$PWD'/../var_defns.sh'
+export USHdir=`grep USHdir $GLOBAL_VAR_DEFNS_FP | cut -d\' -f2`
+export JOBSdir=`grep JOBSdir $GLOBAL_VAR_DEFNS_FP | cut -d\' -f2`
+export PDY=`grep DATE_FIRST_CYCL $GLOBAL_VAR_DEFNS_FP | cut -d\' -f2 | cut -c1-8`
+export cyc=`grep DATE_FIRST_CYCL $GLOBAL_VAR_DEFNS_FP | cut -d\' -f2 | cut -c9-10`
+export SLASH_ENSMEM_SUBDIR='/'
+export ENSMEM_INDX='000'
+export nprocs='48'
+export FCST_LEN_HRS='12'
```

Comment:
`FCST_LEN_HRS` is in the `var_defns.sh` that is exported earlier in the script. I'd remove this to avoid conflict since forecast hour variable can vary.

Reply:
Done